nextcord/channel.py

```diff
@@ -556,7 +563,7 @@ async def create_webhook(self, *, name: str, avatar: Optional[bytes] = None, rea
         -------------
         name: :class:`str`
             The webhook's name.
-        avatar: Optional[:class:`bytes`]
+        avatar: Optional[Union[:class:`bytes`, :class:`Asset`, :class:`Attachment`, :class:`File`]]
             A :term:`py:bytes-like object` representing the webhook's default avatar.
```

Comment:
Docuemntation such as this should be fixed throughout the PR,
L566 says it accepts bytes/asset/attachment/file, but it's description does not match that